templates/components/modals/objective_form.html

```diff
@@ -35,10 +35,13 @@
           class="hikaya-select"
           inputId="parent"
           name="parent"
+          label="name"
           :searchable="false"
           v-model="parent_id"
           placeholder="Choose a parent objective"
-          :options="parent_obj_list.map(objective => ({value: objective.id, label: objective.name}))">
+          :options="parent_obj_list"
+          :reduce="objective => objective.id"
```

Comment:
nice!! I need to learn this reduce method. looks super clean! 😄

Reply:
The `reduce`  property is unique to the `v-select` component however it should work the same way as the `reduce` method for JS.